packages/grid/src/styled/splitter/StyledSeparator.ts

```diff
@@ -0,0 +1,48 @@
+/**
+ * Copyright Zendesk, Inc.
+ *
+ * Use of this source code is governed under the Apache License, Version 2.0
+ * found at http://www.apache.org/licenses/LICENSE-2.0.
+ */
+
+import { retrieveComponentStyles, DEFAULT_THEME, getColor } from '@zendeskgarden/react-theming';
+import styled, { css, ThemeProps, DefaultTheme } from 'styled-components';
+
+const COMPONENT_ID = 'splitter.separator';
+
+interface IStyledSeparatorProps {
+  isHorizontal: boolean;
+}
+
+const sizeStyles = ({ isHorizontal, theme }: IStyledSeparatorProps & ThemeProps<DefaultTheme>) => {
+  const separatorWidth = isHorizontal === false ? theme.borderWidths.sm : '100%';
+  const separatorHeight = isHorizontal ? theme.borderWidths.sm : '100%';
+
+  return css`
+    margin: 0 auto;
+    width: ${separatorWidth};
+    height: ${separatorHeight};
+  `;
+};
+
+const colorStyles = ({ theme }: ThemeProps<DefaultTheme>) => {
+  const greyBackgroundColor = getColor('neutralHue', 300, theme);
+
+  return css`
+    background-color: ${greyBackgroundColor};
+  `;
+};
+
+export const StyledSeparator = styled.div.attrs<IStyledSeparatorProps>({
+  'data-garden-id': COMPONENT_ID,
+  'data-garden-version': PACKAGE_VERSION
+})<IStyledSeparatorProps>`
+  ${sizeStyles}
+  ${colorStyles}
```

Comment:
I'm hoping to take some time to dig into the DOM structure and styling for the `Pane.Splitter`. With the current setup, we run into some crufty styling like this...

<img width="61" alt="Screen Shot 2022-04-15 at 2 47 22 PM" src="https://user-images.githubusercontent.com/143773/163611130-6d46e5f8-2370-4a15-97e6-ca535befe379.png">

I think there is an opportunity to smooth this out – maybe even over-shooting the separator by a pixel so that we avoid focus/hover mismatches like this...

<img width="43" alt="Screen Shot 2022-04-15 at 2 50 00 PM" src="https://user-images.githubusercontent.com/143773/163613215-27a0826e-06ce-41c2-868f-8da571d2afd2.png">

Reply:
I've worked on smoothing this out in https://github.com/zendeskgarden/react-components/compare/splitter/implementation...jzempel/splitter

Reply:
Thanks again for the help on these fixes. Just merged this branch in 💯